test/java/seedu/addressbook/commands/SortCommandTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.util.TestUtil;
+import seedu.addressbook.util.TypicalPersons;
+
+import java.util.List;
+
+import static org.junit.Assert.assertTrue;
+
+public class SortCommandTest {
+
+    @Test
+    public void execute() throws Exception {
```

Comment:
You have a redundant `throws Exception` here.

Reply:
Fixed